cdap-ui/cdap-ui-upgrade/index.js

```diff
@@ -38,12 +38,18 @@ function checkPreRequisite() {
   if (!fs.existsSync(newuizippath)) {
     throw 'UI Pack archive does not exists at the location \''+ newuizippath + '\'. Please check the path specified.';
   }
+  if (newuizippath.indexOf('.zip') !== newuizippath.length - 4) {
```

Comment:
I don't think this check is 100% safe since indexOf finds the first occurrence of `.zip`. There could be a folder with xyz.zipper.zip and this check will fail. Might be better to substring the last 4 characters of `newuizippath`